### ISSUE TITLE
USABILITY: subscribed collections

### ISSUE THREAD

**Issue by tmazuryk** [Cores-ts/Fundingbox-support#10]
My assumption is that when I subscribe to a collection, its' items will show up to me when I start exploring. I don't have a feeling this is a case....

**Comment by xurxo** [comment]
Yes, thats the idea, but this content will show up in the discovery section, which it's not available yet.
We'll open the discovery part as soon we have improved the data, hopefully on the next release, next week.

**Comment by tmazuryk** [comment]
OK, got it!
